bool NewTarget
int32 ControlGroup
float64 Period
geometry_msgs/Transform TargetTransform
float64 TargetElbowAngle
---
bool inProgress
bool isReachable
bool isAccepted
